src/platforms/ruby/common/configuration/options.mdx

```diff
@@ -1,16 +1,16 @@
 ---
-title: Configuration
-sidebar_order: 200
+title: Basic Options
```

Comment:
@HazAT - it looks like the common file for all platforms hasn't been adopted here; can you review to see why not? We document its use here - https://docs.sentry.io/contributing/approach/write-configuration/#basic-options

Reply:
I intentionally did this after went through the shared options file and found there are about 50% of the config options are Ruby-specific, like

- async
- backtrace_cleanup_callback
- breadcrumbs_logger
- enabled_environments
- excluded_exceptions (this is not the same as the `in-app-exclude` option)
- inspect_exception_causes_for_exclusion
- logger

which are 8 of the total 15 options. so I figure it would be better to have a new for it